src/blog/2024/08/comparing-dashboard-2-with-uibuilder.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Node-RED Dashboard 2.0 vs UI-Builder: A Comprehensive Comparison"
+subtitle: Understanding the Differences Between Node-RED Dashboard 2.0 and UI-Builder
+description: Compare Node-RED Dashboard 2.0 and UI-Builder. Discover their installation ease, customization, performance, and support to find the best solution for your needs. 
+date: 2024-08-31
+authors: ["sumit-shinde"]
+image: 
+tags:
+    - node-red dashboard
+    - ui-builder
+    - comparison
+    - dashboard
+
+---
+
+When choosing a dashboard solution for Node-RED, two popular options are Node-RED Dashboard 2.0 and UI-Builder. This article compares these tools across several key areas, including installation, ease of use, development activity, and customizability, to help you decide which one best suits your needs.
+
+<!--more-->
+
+## How Easy is it to Install?
+
+### Node-RED Dashboard 2.0
+
+When searching for `flowfuse/node-red-dashboard` on Google, the first result we get to the documentation, which is useful. However, in the Node-RED Palette Manager, finding the correct package can be confusing because Node-RED Dashboard 2.0 itself publishes some nodes and plugins to help the users. This can make it bit confusing for new users to locate the right package.
+
+### UI-Builder
+
+Finding UI-Builder is straightforward, as its package name `node-red-ui-builder` is distinct and easily identifiable both on Google and in the Node-RED Palette Manager. This clear naming helps users quickly locate the correct package.
+
+## How Easy is it to Get Started?
+
+## Node-RED Dashboard 2.0
+
+Getting started with Node-RED Dashboard 2.0 is relatively easy due to its low-code approach. It features a sidebar for managing UI widgets, themes, configurations, and settings, with intuitive navigation to the dashboard page. This makes it accessible for users with varying levels of technical expertise.
+
+### UI-Builder
+
+UI-Builder can be more challenging to start with, as it does not follow a low-code approach and lacks a separate sidebar for managing UI elements. Navigating the dashboard built with UI-Builder can be less straightforward, requiring users to have a deeper understanding of coding and UI design principles.
+
+## How Easy is it to Migrate from Node-RED Dashboard ?
```

Comment:
Worth adding a paragraph here of context on what Node-RED Dashboard is, new users of Node-RED will not know.